doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,351 @@
+19) [Fiware-Service](#19-fiware-service)
+20) [Fiware-Servicepath](#20-fiware-servicepath)
+21) [Fiware-Total-Count](#21-fiware-total-count)
+22) [Fiware-Correlator](#22-fiware-correlator)                                     
+
+23) [X-Auth-Token](#23-x-auth-token)
+
+
+
+# HTTP headers used by Orion with their description
+
+
+
+## 1. Accept
+
+Accept HTTP header is used in requests. 
+Accept HTTP header is used in the request to specify which MIME type the client accepts. Typically its value is `application/json` (for JSON MIME type) although some operations in the NGSIv2 API also allow `text/plain`. 
+For example in request URL if we use:
+	
+    curl ... -H 'Accept: application/json'
+		 
+On this request we'll get the response data in well defined JSON format.
+
+The API response payloads in this specification are based on `application/json` and  (for attribute value 
+type operation) `text/plain` MIME types. Clients issuing HTTP requests with accept types different 
+than those will get a `406 Not Acceptable` error.
+
+[Top](#top)
+
+## 2. Content-Length
+
+This header is used by both requests and responses.
+Content-length HTTP header is a mandatory header in Orion responses.
+It defines the length of the request and response body in bytes.
+Orion Context Broker expects always a Content-Length header in all client requests, otherwise the client will receive a `411 Length Required` response. 
+This is due to the way the underlying HTTP library (libmicrohttpd) works.
+	
+    Content-Length: 34	
+
+[Top](#top)
+ 
+## 3. Content-Type 
+
+It is used to specify the MIME type of the request or response. Typically its value is `application/json` (for JSON MIME type) although some operations in the NGSIv2 API also allow `text/plain`.
+
+    curl ... -H 'Content-Type: application/json'
+
+This request will send data in well defined JSON format.
+
+[Top](#top)
+ 
+## 4. Connection
+ 
+ This header is used by both requests and responses.
+ Orion is not currently doing anything with this header (apart from checking if its value is `close` to print a log trace).
+	
+    Connection: Keep-Alive
+    	
+[Top](#top)	
+  
+## 5. Expect
+
+Expect HTTP header is used in requests.
+It Indicates that behaviour of particular server are required by the client.
+It is typically used when sending a large request body. We expect the server to return back `100 Continue` HTTP status if it can handle the request, or `417 Expectation Failed` if not.
+
+    Expect: 100-continue
+	
+[Top](#top)	
+	
+## 6. Host
+
+This HTTP header is used in Orion responses.
+When we make any request then the corresponding Notification will come on accumulator server, this header gives the port number of accumulator server. 	
+	
+    Host: localhost:1028
+		
+[Top](#top)
+
+## 7. Origin
+
+Origin is used as request HTTP header in Orion. 
+It initiates a request for cross-origin resource sharing (asks server for Access-Control- response fields).
```

Comment:
```suggestion
Its operation is related with CORS, see specific documentation at [../user/cors.md].
```

Thus we can keep the description simple and provide the user a pointer to the place with the topic is explained in detail.

Please, apply the same idea to the rest of CORS-related headers.

Reply:
Fixed in 5bb0337